packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -32,6 +33,12 @@ const propTypes = {
    * Sets focus on content when set to `true`. Focus on content helps in scrolling  within container when there is no interactive element to focus within container.
    */
   setFocusOnContainer: PropTypes.bool,
+  /**
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue)
+   * set the border color to white or not to match the color contast ratios if the content has dark background (only to be used when setFocusOnContainer is used),
+   * when borderColor is not defined or prop is not used, it will used the default border color according to the theme.
+   */
+  borderColor: PropTypes.oneOf(['dark-border', 'light-border', 'default-border']),
```

Comment:
what is the difference between dark-border and default ? I see same colors being used for both.

Reply:
When the borderColor is set to dark, it will always be black irrespective of theme change, when it is default it turns white for lowlight theme and black for fusion and default theme